ncm-systemd/src/main/resources/regular/tests/profiles/unitfile.pan

```diff
@@ -69,6 +69,18 @@ bind "/unitfile" = systemd_unitfile_config[];
         'SocketGroup', 'pipegroup',
         'SocketMode', '660',
         ),
+    'path', dict(
```

Comment:
you still need to add the expected output of this section to `ncm-systemd/src/main/resources/regular/tests/regexps/unitfile0/value`